experiments/swdt/cmd/setup.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"swdt/pkg/config"
+	"swdt/pkg/connections"
+	"swdt/pkg/pwsh/setup"
+
+	"github.com/spf13/cobra"
+)
+
+var configFile, password string
+
+func init() {
+	rootCmd.AddCommand(setupCmd)
+	setupCmd.Flags().StringVarP(&configFile, "config", "c", "samples/config.yaml", "Configuration file path.")
+	setupCmd.Flags().StringVarP(&password, "password", "p", "", "SSH Password -- should read from stdin.")
+}
+
+// setupCmd represents the setup command
+var setupCmd = &cobra.Command{
+	Use:   "setup",
```

Comment:
On reflection, it does not matter at this point. I think the CLI commands set will grow into subcommands anyway.

Reply:
for now, this only starts some auxiliary services (choco, vim, grep, enable rdp), which could be moved to an `initialize` or `init` subcommand; my feeling is that it will be part of something else later.

Reply:
> for now, this only starts some auxiliary services (choco, vim, grep, enable rdp)

...I'll try to add disabling the Firewall :-)